ENIGMAsystem/SHELL/Universal_System/Extensions/DataStructures/data_structures.cpp

```diff
@@ -731,10 +731,10 @@ std::string ds_grid_write(const unsigned int id)
 
       // Write type
       ss.width(2);
-      ss << (unsigned int)((vari.type == enigma::vt_real) ? 0x00 : 0x01);
+      ss << (unsigned int)((vari.type == enigma_user::ty_real) ? 0x00 : 0x01);
```

Comment:
We're already in namespace `enigma_user`, so these are technically unnecessary. But they are harmless.

Reply:
Fixed in commit 2b30bd52ef6c44f9ae502243213787d6311641ae